.github/workflows/python_unit_test.yaml

```diff
@@ -7,6 +7,10 @@ on:
         description: 'A URL identifying a Bash script that can run commands needed for service-specific test setup'
```

Comment:
imo this repo-specific test file should be stored in the repo, not here

Reply:
I don't think there should be anything repo-specific. The whole idea was to make a configurable standard workflow. Install poetry, poetry install, run pytest. Maybe configure whether postgres is needed, maybe configure pytest command :man_shrugging:

Reply:
maybe also optionally configure the working dir, for example in `gen3authz` the poetry files and tests files are in a subdir: https://github.com/uc-cdis/gen3authz/blob/dcef644/.travis.yml#L10

Reply:
So, I think there will be repo-specific steps for all users of this, just because the tests are all set up slightly differently across services. That's what the dependency-file-location argument is meant to enable. That being said, I wasn't sure on whether storing those here or in the repo makes more sense, so if you think putting that in the service repo will be a better idea, that sounds good to me.

Reply:
I have made this an optional item so you can add an input variable to a setup script but if it's not needed then we just skip.

Reply:
I also made Postgresql a optional input boolean.

Reply:
I'm not sure this URL will work reliably for the setup script on branches? Or I guess the person writing the workflow that uses this template will need to make sure the URL is dynamic based on where the PR is coming from